src/app/services/navigation.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import AppValues from "../common/app.values";
+import ProjectLinks from "../project-links";
+
+@Injectable()
+export class NavigationService {
+    constructor(private router: Router) {}
+
+    public navigateToHome(params?: object): void {
```

Comment:
As these parameters in all methods are associated with particular route, can we make specific arguments instead of `object` ?

Reply:
In several cases, these are different parameters and we do not know what will be in the future

Reply:
Give me an example, where same route has different parameter kinds

Reply:
Added srecific arguments instead of object